nni/compression/pytorch/speedup/error_code.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# Error Code of the speedup
+class SpeedupError(Exception):
+    def __init__(self, msg):
+        self.msg = msg
+
+    def __str__(self):
+        return str(self.msg)
+
+class EmptyLayerError(SpeedupError):
+    def __init__(self):
+        super(EmptyLayerError, self).__init__("Pruning a Layer to empty is not legal")
```

Comment:
In this situation, should we suggest the user add this layer in `exclude`?

Reply:
Not exactly the same,  there may be two reasons that a layer is pruned to empty: (1) it's sparsity ratio equals to 1.0 (2) it's output is useless after the mask propagation.

Reply:
got it